cupy/cuda/__init__.py

```diff
@@ -18,12 +17,12 @@
 
 _available = None
 
-cub_enabled = False  # default to not use CUB for backward compatibility
 # TODO(leofang): always import cub (but not enable it) when hipCUB is supported
 if not runtime.is_hip:
     from cupy.cuda import cub  # NOQA
-    if int(os.getenv('CUB_DISABLED', 1)) == 0:
-        cub_enabled = True
+    cub_enabled = True
+else:
+    cub_enabled = False
```

Comment:
I changed `cub_enabled` to set `True` default because from this PR the routines refer `_accelerator._routine_accelerators` to determine if it uses `cub` or not.
cc: @leofang

Reply:
Now this variable just reflects if the users can use `cupy.cuda.cub.*` functions as the thin wrapper.

Reply:
Thanks!